content/webapp/pages/article.tsx

```diff
@@ -225,7 +232,13 @@ const ArticlePage: FC<Props> = ({ article, jsonLd }) => {
                   </Space>
                 </Fragment>
               ))}
-
+            {readingTime && isArticleorSerialFormat ? (
+              <span className={font('intr', 6)}>
+                <span className={font('intr', 5)}> | </span>
```

Comment:
Could we wrap this in a `<Fragment>`, like the other contributors? I think that might tidy up the uneven spacing – adding the vertical bar ourselves feels odd.